test/e2e/e2e_suite_test.go

```diff
@@ -176,6 +173,26 @@ func applyManifests(kindCluster kind.Cluster, manifests *string) {
 	kindCluster.ApplyYAML(*manifests)
 }
 
+func deployCAPIComponents(kindCluster kind.Cluster) {
+	fmt.Fprintf(GinkgoWriter, "Generating CAPI manifests\n")
+
+	// Build the manifests using kustomize
+	capiManifests, err := exec.Command(*kustomizeBinary, "build", "https://github.com/kubernetes-sigs/cluster-api//config/default").Output()
```

Comment:
Any concern about reading all of this into memory vs using `StdoutPipe` and an `io.Copy`?

Reply:
Maybe, but this is only temporary until we publish capi manifests